internal/template/mapper/mapper.go

```diff
@@ -119,6 +121,17 @@ func injectDynamicResString(val string) (string, error) {
 		val = strings.Replace(val, dynamicValue, parsedDynamicValue, 1)
 	}
 
+	// If the value is a relative path, we need to convert it to an absolute path.
+	isPath := regexp.MustCompile(`^.\/(.*)$`).MatchString(val)
```

Comment:
this can be optimized by moving the MustCompile to a package variable, to avoid the compilation to happen over and over again